rllib/env/single_agent_env_runner.py

```diff
@@ -820,12 +620,15 @@ def make_env(self) -> None:
             )
         gym.register("rllib-single-agent-env-v0", entry_point=entry_point)
 
-        # Wrap into `VectorListInfo`` wrapper to get infos as lists.
-        self.env: gym.Wrapper = gym.wrappers.VectorListInfo(
-            gym.vector.make(
+        self.env = DictInfoToList(
+            gym.make_vec(
                 "rllib-single-agent-env-v0",
                 num_envs=self.config.num_envs_per_env_runner,
-                asynchronous=self.config.remote_worker_envs,
+                vectorization_mode=(
```

Comment:
Very nice! I remember in my problems with the update nto gymnasium==1.0.0 this was one reason of failure. It needs now this mode.